src/backend/binaryen/utils.ts

```diff
@@ -1116,6 +1116,58 @@ export namespace FunctionalFuncs {
             case ts.SyntaxKind.GreaterThanEqualsToken: {
                 return module.f64.ge(leftValueRef, rightValueRef);
             }
+            case ts.SyntaxKind.GreaterThanGreaterThanToken: {
+                return convertTypeToF64(
+                    module,
+                    module.i32.shr_s(
+                        convertTypeToI32(
+                            module,
+                            convertTypeToI64(
+                                module,
+                                leftValueRef,
+                                binaryen.f64,
+                            ),
+                            binaryen.i64,
+                        ),
```

Comment:
Why we need to covert to i64 then covert to i32?

Reply:
Here we convert `number` to `i32` by using `i32.trunc_s`, so if the range of the source larger than 2^31-1，the runtime will throw an integer overflow error.